cmd/apiserver/main.go

```diff
@@ -14,13 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-
 package main
 
 import (
+	"flag"
+
 	// Make sure glide gets these dependencies
-	_ "k8s.io/apimachinery/pkg/apis/meta/v1"
 	_ "github.com/go-openapi/loads"
+	_ "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
this import order looks funny -- github then k8s.io then github then k8s.io. Can you group the two github ones and the two k8s.io ones in two blocks?

Reply:
Sure - done!